readme.md

```diff
@@ -35,12 +35,11 @@ Once you do a search, you should see a list similar to the one below (versions m
 | --- | --- |
 | Microsoft.Toolkit | .NET Standard NuGet package containing common code |
 | Microsoft.Toolkit.HighPerformance | .NET Standard and .NET Core NuGet package with performance oriented helpers, extensions, etc. |
-| Microsoft.Toolkit.Parsers | .NET Standard NuGet package containing cross-platform parsers, such as Markdown and RSS |
-| Microsoft.Toolkit.Services | .NET Standard NuGet package containing cross-platform services |
+| Microsoft.Toolkit.Parsers | .NET Standard NuGet package containing cross-platform parsers, such as Markdown |
```

Comment:
I know we haven't gotten there yet, but should we mark the current Markdown parser as Obsolete at least now?

If we don't migrate the UI component soon, we could internalize all of this code just for the control usage... right?